### PR TITLE
chore(documentation): update CSS card-control styles and merge docs pages with the one from the web-component

### PR DESCRIPTION
**Pull request by oliverschuerch**

### CODE REVIEW

File: packages/documentation/src/stories/components/forms/card-control/standard-html/card-control-firefox-fallback.sample.ts [hunk 1]
```diff
@@ -0,0 +1,12 @@
+document.addEventListener('input', e => {
```
Comment (imagoiq) [inline packages/documentation/src/stories/components/forms/card-control/standard-html/card-control-firefox-fallback.sample.ts:1]: Just by curiosity, did you move the file or recreate it and copy and paste the content or with another method? Github sees it as a new file, not as a move.

File: packages/documentation/tsconfig.json [hunk 2]
```diff
@@ -1,5 +1,8 @@
 {
   "compilerOptions": {
+    "paths": {
+      "@/*": ["./src/*"]
+    },
```
Comment (imagoiq) [inline packages/documentation/tsconfig.json:5]: Good idea, we should do it in all other places, can you open an issue, please?

File: packages/documentation/types/storybook.ts [hunk 3]
```diff
@@ -1,13 +1,13 @@
 import { Args, Meta } from '@storybook/web-components';
 import { PackageType } from './package';
 
+export type TagPackagePrefix<K> = K extends string ? `package:${K}` : K;
+
```
Comment (imagoiq) [inline packages/documentation/types/storybook.ts:5]: Why this move? I don't really care, but is it your intention or the tooling?

**Comment by oliverschuerch** [comment]
> Radio type is quite different from the following PR (we should discuss what is the right way): #2774 https://preview-2774--swisspost-design-system-next.netlify.app/?path=/docs/d83829b2-7de2-48d2-be64-07a80c9caef3--docs#validation-messages
> 
> On hover, the SelectedItem color is removed, is it correct?
> 
> Disabled style is as well different from (color + inner circle is filled with white in your version) : #2813 https://preview-2813--swisspost-design-system-next.netlify.app/?path=/docs/151242aa-a074-4a55-a81c-db597c83cdad--docs
> 
> On Firefox, the contrast with the label is not sufficient: firefox_b0loouslkD
> 
> Maybe we can talk about all this in the roundtable?

I think I found a satisfying solution for the hcm styles. Please have a look :grinning:

File: pnpm-lock.yaml [hunk 4]
```diff
@@ -561,7 +561,7 @@ importers:
         version: 7.6.17(lit@3.1.2)(react-dom@18.2.0)(react@18.2.0)
       '@storybook/web-components-vite':
         specifier: 7.6.17
-        version: 7.6.17(lit@3.1.2)(react-dom@18.2.0)(react@18.2.0)(typescript@5.3.3)(vite@5.0.12)
+        version: 7.6.17(lit@3.1.2)(react-dom@18.2.0)(react@18.2.0)(typescript@5.3.3)(vite@5.1.5)
```
Comment (imagoiq) [inline pnpm-lock.yaml:564]: This will probably resurface the issue that we had last week

**Review by imagoiq — CHANGES_REQUESTED** [review]
In HCM on Firefox and hover state, the check has not enough contrast and the border of the checkbox is not visible anymore.

firefox_HDus3PCFNF

**Comment by oliverschuerch** [comment]
To fix the hcm styles, I've created a new ticket: #2982

**Review by gfellerph — APPROVED** [review]

**Review by imagoiq — APPROVED** [review]

**Pull request merged**